### PR TITLE
ACT-1167 - Add schema support to policies and entity types

### PR DESCRIPTION
**Pull request by smbecker**

### CODE REVIEW

**Review by mmitoff — APPROVED** [review]

File: src/sdk/resources/policies.ts [hunk 1]
```diff
@@ -13,6 +13,11 @@ export class PoliciesResource extends PagedResource<Policy> {
 		return new PolicyTrackingGroupsResource(this.httpClient, policyId);
 	}
 
+	async schema(id: string, token?: CancellationToken): Promise<PropertySchema> {
+		const result = await this.httpClient.get(`${this.resourcePath}/${id}/schema`, null, token);
```
Comment (jtirjan) [inline src/sdk/resources/policies.ts:17]: Passing `null` for `params` causes `http-client` to bomb

```suggestion
		const result = await this.httpClient.get(`${this.resourcePath}/${id}/schema`, {}, token);
```

image

**Review by jtirjan — CHANGES_REQUESTED** [review]

File: src/sdk/resources/policies.ts (hunk 1 above)
Comment (jtirjan) [reply src/sdk/resources/policies.ts:17]: When attempting to destructure `params`
image

File: src/sdk/resources/entityTypes.ts [hunk 2]
```diff
@@ -13,6 +13,11 @@ export class EntityTypesResource extends PagedResource<EntityType | BlockListEnt
 		return getTypedResponse<EntityType[]>(result);
 	}
 
+	async schema(id: string, token?: CancellationToken): Promise<PropertySchema> {
+		const result = await this.httpClient.get(`${this.resourcePath}/${id}/schema`, null, token);
```
Comment (jtirjan) [inline src/sdk/resources/entityTypes.ts:17]: This one as well
```suggestion
		const result = await this.httpClient.get(`${this.resourcePath}/${id}/schema`, {}, token);
```

File: src/sdk/resources/policies.ts (hunk 1 above)
Comment (smbecker) [reply src/sdk/resources/policies.ts:17]: Should be good now.

**Review by jtirjan — APPROVED** [review]

**Pull request merged**